### PR TITLE
Support MsgResetAccount

### PR DESCRIPTION
**Pull request by mikluke**

### CODE REVIEW

File: internal/consumer/blockchain/blockchain.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		})
+	}
+}
+
+func processMsgResetAccount(ctx context.Context, is storage.IndexStorage, fs storage.FileStorage, msg pdv.MsgResetAccount) error {
+	if err := is.DeleteProfile(ctx, msg.AccountOwner.String()); err != nil {
+		return fmt.Errorf("failed to delete profile: %w", err)
+	}
+
+	if err := fs.DeleteData(ctx, msg.AccountOwner.String()); err != nil {
```
Comment (megaherz) [inline internal/consumer/blockchain/blockchain.go:104]: It is likely to be long. Perhaps it makes sense to run is async, i.e. in goroutine and log err if any.

File: internal/service/service.go [hunk 2]
```diff
@@ -295,11 +295,11 @@ func getSetProfileParams(owner sdk.AccAddress, p schema.V1Profile) *storage.SetP
 }
 
 func getPDVOwnerPrefix(owner string) string {
-	return fmt.Sprintf("pdv/%s", owner)
+	return fmt.Sprintf("%s/pdv", owner)
```
Comment (megaherz) [inline internal/service/service.go:298]: Breaking change right? We need to keep in mind as far as going to crease some dataset samples.

File: internal/storage/postgres/postgres.go [hunk 3]
```diff
@@ -113,6 +166,15 @@ func (s pg) SetProfile(ctx context.Context, p *storage.SetProfileParams) error {
 	return nil
 }
 
+// DeleteProfile deletes profile.
+func (s pg) DeleteProfile(ctx context.Context, addr string) error {
+	if _, err := s.ext.ExecContext(ctx, `DELETE FROM profile CASCADE WHERE address = $1`, addr); err != nil {
```
Comment (megaherz) [inline internal/storage/postgres/postgres.go:171]: Why CASCADE? Groundwork for the future?

Comment (mikluke) [reply internal/storage/postgres/postgres.go:171]: copy-paste

**Review by megaherz — APPROVED** [review]

**Pull request merged**